saleor/static/dashboard-next/siteSettings/components/SiteSettingsDetails/SiteSettingsDetails.tsx

```diff
@@ -0,0 +1,92 @@
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import TextField from "@material-ui/core/TextField";
+import * as React from "react";
+
+import CardTitle from "../../../components/CardTitle";
+import FormSpacer from "../../../components/FormSpacer";
+import i18n from "../../../i18n";
+import { SiteSettingsPageFormData } from "../SiteSettingsPage";
+
+interface SiteSettingsDetailsProps {
+  data: SiteSettingsPageFormData;
+  errors: Partial<{
+    description: string;
+    domain: string;
+    name: string;
+  }>;
+  disabled: boolean;
+  onChange: (event: React.ChangeEvent<any>) => void;
+}
+
+const SiteSettingsDetails: React.StatelessComponent<
+  SiteSettingsDetailsProps
+> = ({ data, disabled, errors, onChange }) => (
+  <Card>
+    <CardTitle
+      title={i18n.t("General Information", {
+        context: "card title"
+      })}
+    />
+    <CardContent>
+      <TextField
+        disabled={disabled}
+        error={!!errors.name}
+        fullWidth
+        name="name"
+        label={i18n.t("Store Name", {
```

Comment:
"Store name" is not a great label as it can be read as "save the name" (including by translators), something more friendly like "Name of your store" or "Your brand name" could work better.